Encounter1/Resources/values/styles.xml

```diff
@@ -1,11 +1,18 @@
 <resources>
 
-    <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.MaterialComponents.Light.DarkActionBar">
-        <!-- Customize your theme here. -->
-        <item name="colorPrimary">@color/colorPrimary</item>
-        <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
-        <item name="colorAccent">@color/colorAccent</item>
-    </style>
+	<style name="EncounterMe.LightMode" parent="@style/Theme.AppCompat.NoActionBar">
+		<item name="android:colorBackground">@color/encounter_light_background</item>
+		<item name="android:windowBackground">@color/encounter_light_background</item>
+		<item name="android:colorAccent">@color/encounter_accent_1</item>
+		<item name="android:colorPrimary">@color/encounter_light_text</item>
```

Comment:
Same problem with menu and username as with locationList.
![whiteMenu](https://user-images.githubusercontent.com/50084966/134889683-a88e87c5-5cad-45b5-9c44-ff9b1827c3fb.PNG)

Reply:
Yes, it's like that on purpose. I haven't applied styling properties to them yet.

Reply:
Rewrote the code a bit, so the text is dark once again :)